app-feedback-program.md

```diff
@@ -0,0 +1,20 @@
+# Electron App Feedback Program
+The App Feedback Program is for large-scale Electron apps to report app-specific bugs, blockers, and feature requests to the Electron team. There is an #app-feedback-program Slack channel for apps to collaborate together in. Electron is working on building a streamlined release process and having faster releases. If you would like to participate, please send us a message at info@electronjs.org.
+
+## Who can join?
+This is a new program for Electron and a work in progress. Please bear with us as we refine our process. Our initial criteria and expectations for joining this program include the following items:
+- Test your app during the beta period for 10,000+ user-hours. For example, 10 user-hours could be 10 people testing for one hour, or one person testing for 10 hours.
+- Have a single point-person to speak for the app (e.g., report Electron bugs and blockers) and check-in weekly, either during our regularly scheduled meeting, or asynchronously with @sofianguy
+- Your app should run for 10,000+ user-hours during the beta cycle. We are looking to release stable versions every 2-3 months. For example, you can test 5,000 user-hours on 3.0.0-beta2 and then 5,000 user-hours on 3.0.0-beta5. More is better, but we understand some applications cannot test every beta release.
+- You are willing to share the following information listed in the next question.
+
+## What info does my Electron app have to share?
+- Total user-hours your app has been running with any beta release. CI or QA hours do not count towards the total, however internal releases do count.
+- Version of Electron that your app is testing with (e.g., 3.0.0-beta3)
+- Any bugs preventing your application from upgrading to the release line being beta tested
+
+## Ok cool, but why should my Electron app join?
```

Comment:
nit: for consistency we may want this to simply be: `Why Should My Electron App Join?` and capitalize the others as headers as well :)